src/TKey.cpp

```diff
@@ -71,6 +71,7 @@ void TKey::setName(const QString& name)
     mpHost->getKeyUnit()->mLookupTable.insertMulti(name, this);
 }
 
+// This method is potentially flawed in that only the first child match is ever executed...!
```

Comment:
However the lua function elsewhere that counts how many of various mudlet types match a given type and name could tell the user that they have several keys matching a particular name and they would not necessarily know that only one will be used - further more - since key-bindings (like triggers and aliases) are matched by (in this case) a regex of input text and the name is not relevant it is not entirely clear how useful it would be to match by name (especially if a package/module comes along and supplants an existing key-binding...!)

This strikes me as a little flawed.... :slightly_frowning_face: